examples/toast-message/app/message.server.ts

```diff
@@ -0,0 +1,35 @@
+import { createCookieSessionStorage } from "remix";
+
+const ONE_YEAR = 1000 * 60 * 60 * 24 * 365;
+
+const { commitSession, getSession } = createCookieSessionStorage({
+  cookie: {
+    name: "__message",
+    path: "/",
+    httpOnly: true,
+    sameSite: "lax",
+    expires: new Date(Date.now() + ONE_YEAR),
+    secrets: ["SUPER_SECRET"],
+    secure: true
+  }
+});
+
+const setSuccessMessage = async (request: Request, message: string) => {
```

Comment:
I would make these functions receive the session object so you share a session created before calling them, which usually happens at the end of the action function.

Also, a little nitpicking, they could be created using the function keyword instead of an arrow 😛